Add memory usage warning in quick-start documentation (#15967)
mistercrunch/Airflow: docs/apache-airflow/start/docker.rst

See:ref:`Docker Compose environment variables <docker-compose-env-variables>`

.. warning::

>     Default amount of memory available for Docker on ``MacOS`` is often not enough to get Airflow up and running.

Reviewer: ```suggestion
    Default amount of memory available for Docker on MacOS is often not enough to get Airflow up and running.
```

>     Default amount of memory available for Docker on ``MacOS`` is often not enough to get Airflow up and running.

Reviewer: I am not sure this is the right place for this warning. What do you think to move this to the "Before you begin"  section?

Author: Yep. looks better

    If you have not enough memory available it might lead to airflow webserver continuously restarting.
    You should have at least 4GB memory allocated for the Docker Engine (ideally 8GB). You can check
    and change the amount of memory in `Resources <https://docs.docker.com/docker-for-mac/#resources>`_

>     You can alsocheck if you have enough memory by running this command:

Reviewer: ```suggestion
    You can also check if you have enough memory by running this command:
```

    .. code-block:: bash

>         docker run --rm --entrypoint /bin/bash "debian:buster-slim" -c 'numfmt --to iec $(echo $(($(getconf _PHYS_PAGES) * $(getconf PAGE_SIZE))))'

Reviewer: ```suggestion
        docker run --rm "debian:buster-slim" bash -c 'numfmt --to iec $(echo $(($(getconf _PHYS_PAGES) * $(getconf PAGE_SIZE))))'
```
A little easier.

Reviewer: ````bash
$ docker run --rm  "debian:buster-slim" bash -c 'numfmt --to iec $(echo $(($(getconf _PHYS_PAGES) * $(getconf PAGE_SIZE))))'
5.9G
````
